api/database/seeds/ArticleSeeder.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class ArticleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        factory(App\Models\Article::class, 10)->create();
+    }
```

Comment:
yes, that what I was planning to do after architecture fixes merge, to check everything from the start, with all migrations and seeding